build-arm-mingw32ce/build-mingw32ce.sh

```diff
@@ -173,7 +173,7 @@ fi
 components=`echo "${components}" | sed -e 's/[ 	,][ 	,]*/,/g' -e 's/,$//'`
 
 # Don't build in source directory, it will clobber things and cleanup is hard.
-if [ -d ${BUILD_DIR}/.svn ]; then
+if [ -d ${BUILD_DIR}/.svn ] || [ -e ${BUILD_DIR}/.gitignore ]; then
```

Comment:
The `.svn` check can be removed, we'll never use Subversion again; and this should better do `-d .git` instead of `-e .gitignore`.